scripts/scr_powers_new/scr_powers_new.gml

```diff
@@ -1,56 +1,57 @@
-function scr_powers_new(argument0, argument1) {
+function scr_powers_new(){
 
-	// Argument0: Company
-	// Argument1: Marine ID
+	// company: Company
+	// marine_number: Marine ID
 
 	// This script checks if the marine is capable of using psychic powers, and if so, assigns them based on experience
 
 	var random_learn;
 	random_learn=false;
 
-	var let,letmax,powers_should_have,powers_have;
-	let="";letmax=0;
+	var powers_should_have,powers_have;
+	var power_code="";letmax=0;
 
-	if (obj_ini.psy_powers="default"){let="D";letmax=7;}
-	if (obj_ini.psy_powers="biomancy"){let="B";letmax=5;}
-	if (obj_ini.psy_powers="pyromancy"){let="P";letmax=5;}
-	if (obj_ini.psy_powers="telekinesis"){let="T";letmax=5;}
-	if (obj_ini.psy_powers="rune Magick") or (obj_ini.psy_powers="Rune Magick"){let="R";letmax=5;}
+	if (obj_ini.psy_powers="default"){power_code="D";letmax=7;}
+	if (obj_ini.psy_powers="biomancy"){power_code="B";letmax=5;}
+	if (obj_ini.psy_powers="pyromancy"){power_code="P";letmax=5;}
+	if (obj_ini.psy_powers="telekinesis"){power_code="T";letmax=5;}
+	if (obj_ini.psy_powers="rune Magick") or (obj_ini.psy_powers="Rune Magick"){power_code="R";letmax=5;}
```

Comment:
```suggestion
        power_code = string_upper(string_char_at(obj_ini.psy_powers, 1)) //strings start at idx 1 for...reasons...
        letmax = 5;
        if power_code == "D"
            letmax = 7    
```

doing a ton of if's is pain on the eyes, might as well change that part of the system while messing with this stuff.